projects/storefrontlib/src/cms-structure/page/config/default-header.config.ts

```diff
@@ -13,26 +23,94 @@ export const headerComponents: {
     flexType: 'CMSSiteContextComponent',
     context: 'CURRENCY',
   },
-  storeFinder: {
+  StoreFinder: {
     typeCode: 'CMSLinkComponent',
     flexType: 'CMSLinkComponent',
     linkName: 'Find a Store',
     url: '/store-finder',
   },
-  breadcrumbComponent: {
+  BreadcrumbComponent: {
     typeCode: 'BreadcrumbComponent',
     flexType: 'BreadcrumbComponent',
   },
+  Logo: {
+    typeCode: 'SimpleBannerComponent',
+    flexType: 'SimpleBannerComponent',
+    uid: 'logo',
+    media: {
+      mime: 'svg/image/svg+xml',
+      url: 'https://www.sap.com/dam/application/shared/logos/sap-logo-svg.svg',
+    },
+    urlLink: '/',
+  },
+  SearchBox: {
+    typeCode: 'SearchBoxComponent',
+    flexType: 'SearchBoxComponent',
+    uid: 'SearchBoxComponent',
+  },
+  MiniCart: {
+    typeCode: 'MiniCartComponent',
+    flexType: 'MiniCartComponent',
+    uid: 'MiniCartComponent',
+  },
+  LoginComponent: {
+    typeCode: 'LoginComponent',
+    flexType: 'LoginComponent',
+    uid: 'LoginComponent',
+  },
+  CategoryNavigationComponent: {
+    typeCode: 'CategoryNavigationComponent',
+    flexType: 'CategoryNavigationComponent',
+    uid: 'ElectronicsCategoryNavComponent',
+    modifiedTime: '2019-02-07T19:09:51.416Z',
```

Comment:
Do we really need to go in such details for this? Why is the modifiedTime necessary?

Reply:
copy-paste